README.md

```diff
@@ -1,427 +1,96 @@
-
-**Repos:**
-
- - [Secure-Scuttlebutt](https://github.com/ssbc/secure-scuttlebutt) - The core database library. Wraps leveldb with tools for reading, writing to, and replicating feeds.
- - [Scuttlebot](https://github.com/ssbc/scuttlebot) - A secure-scuttlebutt implementation. Includes the database, networking, blob exchange protocol, and command-line tools.
- - [Patchwork](https://github.com/ssbc/patchwork) - A user-friendly application for sharing content over SSB. Embeds scuttlebot.
+ - [Learn About SSB](./learn.md)
+ - Articles
+   - [Design Challenge: Avoiding Centralization and Singletons](./articles/design-challenge-avoid-centralization-and-singletons.md)
+   - [Design Challenge: Sybil Attacks](./articles/design-challenge-sybil-attack.md)
+   - [Desirable Properties for a Secure Channel](./articles/desirable-properties-for-a-secure-channel.md)
+   - [Secure, Private Channels: the Good, the Bad, and the Ugly](./articles/secure-private-channels.md)
+ - Scuttlebot
+   - [Repo & Overview](https://github.com/ssbc/scuttlebot)
+   - [An Introduction to Developing with Scuttlebot](todo)
+   - [API Docs](https://github.com/ssbc/scuttlebot/blob/master/api.md)
+   - Plugins
+     - [Blobs](https://github.com/ssbc/scuttlebot/blob/master/plugins/blobs.md)
+     - [Block](https://github.com/ssbc/scuttlebot/blob/master/plugins/block.md)
+     - [Friends](https://github.com/ssbc/scuttlebot/blob/master/plugins/friends.md)
+     - [Gossip](https://github.com/ssbc/scuttlebot/blob/master/plugins/gossip.md)
+     - [Invite](https://github.com/ssbc/scuttlebot/blob/master/plugins/invite.md)
+     - [Private](https://github.com/ssbc/scuttlebot/blob/master/plugins/private.md)
+     - [Replicate](https://github.com/ssbc/scuttlebot/blob/master/plugins/replicate.md)
+ - Pull-streams
+   - [Repo & Overview](https://github.com/dominictarr/pull-stream)
+   - [A Primer for Pull-streams](todo)
+   - [Pull Sources](https://github.com/dominictarr/pull-stream/blob/master/docs/sources.md)
+   - [Pull Throughs](https://github.com/dominictarr/pull-stream/blob/master/docs/throughs.md)
+   - [Pull Sinks](https://github.com/dominictarr/pull-stream/blob/master/docs/sinks.md)
+
+
+## Glossary
+
+ - [Secure-Scuttlebutt](https://github.com/ssbc/secure-scuttlebutt) (SSB) - A protocol for replicating logs in a decentralized mesh. The SSB library wraps leveldb with tools for reading, writing to, and replicating feeds.
+ - [Scuttlebot](https://github.com/ssbc/scuttlebot) - A secure-scuttlebutt server and client. Includes the database, networking, blob exchange protocol, and command-line tools.
+ - Pub Servers - SSB peers which run on publicly IPs, and provide connectivity and hosting for users on private IPs. Pubs are not privileged, and do not hold special authority in the network. They are not hosts.
+ - Invite codes - Tokens which may be used to command specific Pub servers to follow a user. These are used to join Pubs.
+ - Feeds - a user's stream of JSON data. Also called logs.
```

Comment:
i'd replaces JSON data with messages, as i think messages are a commonly understood concept for what it really is.

also it's important to know that feeds are signed by the user's key (i.e. known to be from the user, even after gossip).